examples/json_logger/log4rs.yaml

```diff
@@ -0,0 +1,9 @@
+appenders:
+  stdout:
+    kind: console
+    encoder:
+      kind: json
+root:
+  level: info
+  appenders:
+    - stdout
```

Comment:
I'm sorry about this, but could you add a newline before EOF here and Cargo.toml? I'll drop a .editorconfig in the root to prevent this in the future.

Reply:
No worries. I'll fix these up.